.golangci.yaml

```diff
@@ -11,23 +11,23 @@ linters:
     - asciicheck
     - bodyclose
     - depguard
-    - dogsled
```

Comment:
nit: for the items removed, mind adding them in the commented out section at the bottom with a quick reason why we aren't using them anymore?

Reply:
Yep!